features/logging_in_to_the_application.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Login to the app
+
+as a user
+in order to use the application
+I would like to login to my account
+
+
+Background:
+Given the following user exists
+|email|password|name|
+|faraz@craftacademy.se|mypassword|Faraz|
+
+Scenario:
+Given I am on the landing page
+When I click on the link "Login"
+And I fill in the "Email" with "faraz@craftacademy.se"
+And I fill in the "Password" with "mypassword"
+When I click on the link "Log in"
+Then I should be on the root page
```

Comment:
I would avoid using internal expressions like `root page` in scenarios. Remember that outside stakeholders might read it and need to understand what you mean.